browser/ui/tabs/split_view_browser_data.h

```diff
@@ -78,9 +81,12 @@ class SplitViewBrowserData : public BrowserUserData<SplitViewBrowserData> {
   std::vector<Tile>::const_iterator FindTile(const tabs::TabHandle& tab);
   std::vector<Tile>::const_iterator FindTile(const tabs::TabHandle& tab) const;
 
+  void Transfer(SplitViewBrowserData* other, std::vector<Tile> tiles);
+
   std::unique_ptr<SplitViewTabStripModelAdapter> tab_strip_model_adapter_;
 
   std::vector<Tile> tiles_;
+  std::vector<Tile> tiles_to_be_attached_to_new_window_;
```

Comment:
transient data for a tab drag should not be stored as persistent data in SplitViewBrowserData